diff --git a/runtest/syscalls b/runtest/syscalls
index 34b47f36b..42d9b226e 100644
--- a/runtest/syscalls
+++ b/runtest/syscalls
@@ -14,7 +14,7 @@ acct01 acct01
 add_key01 add_key01
 add_key02 add_key02
 add_key03 add_key03
-add_key04 add_key04
+#add_key04 add_key04
 
 adjtimex01 adjtimex01
 adjtimex02 adjtimex02
@@ -313,10 +313,10 @@ fork05 fork05
 fork06 fork06
 fork07 fork07
 fork08 fork08
-fork09 fork09
+#fork09 fork09
 fork10 fork10
 fork11 fork11
-fork13 fork13 -i 1000000
+#fork13 fork13 -i 1000000
 fork14 fork14
 
 fpathconf01 fpathconf01
@@ -719,7 +719,7 @@ msgctl04 msgctl04
 msgstress01 msgstress01
 msgstress02 msgstress02
 msgstress03 msgstress03
-msgstress04 msgstress04
+#msgstress04 msgstress04
 msgctl12 msgctl12
 
 msgget01 msgget01
@@ -771,7 +771,7 @@ open01 open01
 open01A symlink01 -T open01
 open02 open02
 open03 open03
-open04 open04
+#open04 open04
 open05 open05
 open06 open06
 open07 open07
@@ -1261,7 +1261,7 @@ sigrelse01 sigrelse01
 
 sigsuspend01 sigsuspend01
 
-sigwaitinfo01 sigwaitinfo01
+#sigwaitinfo01 sigwaitinfo01
 
 socket01 socket01
 socket02 socket02
@@ -1279,8 +1279,8 @@ sockioctl01 sockioctl01
 splice01 splice01
 splice02 seq 1 20000 | splice02
 splice03 splice03
-splice04 splice04
-splice05 splice05
+#splice04 splice04
+#splice05 splice05
 
 tee01 tee01
 tee02 tee02
@@ -1450,7 +1450,7 @@ utime06 utime06
 utimes01 utimes01
 
 # Introduced from Kernel 2.6.22 onwards
-utimensat01 utimensat_tests.sh
+#utimensat01 utimensat_tests.sh
 
 vfork01 vfork01
 vfork02 vfork02
diff --git a/testcases/kernel/fs/fs-bench/fs-bench-test.sh b/testcases/kernel/fs/fs-bench/fs-bench-test.sh
index c2ca767f0..49e23edd8 100755
--- a/testcases/kernel/fs/fs-bench/fs-bench-test.sh
+++ b/testcases/kernel/fs/fs-bench/fs-bench-test.sh
@@ -6,12 +6,15 @@ if [ -d ./00 ] ; then
     echo -n "Clear old files..."
     /bin/rm -fr ./00 >& /dev/null
     echo "done"
+else
+    echo "Create dir"
+    mkdir 00
 fi
 STARTT=`date +%s`
 echo $STARTT
 echo ""
 echo "## Create files "
-time ~/fs-bench/cr
+time create-file
 
 echo ""
 echo "## tar all "
@@ -23,15 +26,15 @@ time chown -R $USER  ./00
 
 echo ""
 echo "## random access"
-time ~/fs-bench/ra  $MAXFILE
+time random-access  $MAXFILE
 
 echo ""
 echo "## Change mode "
 time chmod -R go+rw  ./00
 
 echo ""
-echo "## Random delete and create"
-time ~/fs-bench/radc  $MAXFILE
+echo "## Random delete and create-fileeate"
+time random-access-del-create  $MAXFILE
 
 echo ""
 echo "## Change mode again"
diff --git a/testcases/kernel/fs/fs-bench/fs-bench-test2.sh b/testcases/kernel/fs/fs-bench/fs-bench-test2.sh
index 25c67afe0..6457ef4bc 100755
--- a/testcases/kernel/fs/fs-bench/fs-bench-test2.sh
+++ b/testcases/kernel/fs/fs-bench/fs-bench-test2.sh
@@ -1,6 +1,11 @@
 #! /bin/bash
 
+MAXFILES=150
+let HALFFILES=MAXFILES/2
+
+
 echo "## Start Test"
+echo "# MAXFILES="${MAXFILES}
 date
 date +%s
 if [ -d ./00 ] ; then
@@ -11,12 +16,12 @@ echo $STARTT
 
 echo ""
 echo "## Create files "
-time ~/fs-bench/cr
+time create-files $MAXFILES
 
 echo ""
 echo "## tar all "
-MAXFILE=`tar cBf - 00 | tar tvBf - 2>&1 | tail -n 1 | awk '{print $6;}'| awk -F'/' '{print $4;}'`
-HALFFILE=`echo "obase=F;ibase=F;$MAXFILE/2" | bc`
+#MAXFILE=`tar cBf - 00 | tar tvBf - 2>&1 | tail -n 1 | awk '{print $6;}'| awk -F'/' '{print $4;}'`
+#HALFFILE=`echo "obase=F;ibase=F;$MAXFILE/2" | bc`
 
 echo ""
 echo "## Remove all files and directories"
@@ -25,15 +30,15 @@ echo "## Remove all files and directories"
 echo ""
 echo "## Create half files"
 echo create half files
-time ~/fs-bench/cr  $HALFFILE
+time create-files  $HALFFILES
 
 echo ""
 echo "## Change owner"
-time chown -R $USER  ./00
+time chown -R root  ./00
 
 echo ""
 echo "## random access"
-time ~/fs-bench/ra  $HALFFILE
+time random-access  $HALFFILES
 
 
 echo ""
@@ -42,7 +47,7 @@ time chmod -R go+rw  ./00
 
 echo ""
 echo "## Random delete and create"
-time ~/fs-bench/radc  $HALFFILE
+time random-access-del-create  $HALFFILES
 
 echo ""
 echo "## Change mode again"
diff --git a/testcases/kernel/fs/racer/fs_racer.sh b/testcases/kernel/fs/racer/fs_racer.sh
index aa38e86db..1d62a1691 100755
--- a/testcases/kernel/fs/racer/fs_racer.sh
+++ b/testcases/kernel/fs/racer/fs_racer.sh
@@ -34,37 +34,37 @@ DIR="$TMPDIR/race"
 execute_test()
 {
 [ -e $DIR ] || mkdir $DIR
-./fs_racer_file_create.sh $DIR $MAX_FILES &
-./fs_racer_file_create.sh $DIR $MAX_FILES &
-./fs_racer_file_create.sh $DIR $MAX_FILES &
+fs_racer_file_create.sh $DIR $MAX_FILES &
+fs_racer_file_create.sh $DIR $MAX_FILES &
+fs_racer_file_create.sh $DIR $MAX_FILES &
 
-./fs_racer_dir_create.sh $DIR $MAX_FILES &
-./fs_racer_dir_create.sh $DIR $MAX_FILES &
-./fs_racer_dir_create.sh $DIR $MAX_FILES &
+fs_racer_dir_create.sh $DIR $MAX_FILES &
+fs_racer_dir_create.sh $DIR $MAX_FILES &
+fs_racer_dir_create.sh $DIR $MAX_FILES &
 
-./fs_racer_file_rename.sh $DIR $MAX_FILES &
-./fs_racer_file_rename.sh $DIR $MAX_FILES &
-./fs_racer_file_rename.sh $DIR $MAX_FILES &
+fs_racer_file_rename.sh $DIR $MAX_FILES &
+fs_racer_file_rename.sh $DIR $MAX_FILES &
+fs_racer_file_rename.sh $DIR $MAX_FILES &
 
-./fs_racer_file_link.sh $DIR $MAX_FILES &
-./fs_racer_file_link.sh $DIR $MAX_FILES &
-./fs_racer_file_link.sh $DIR $MAX_FILES &
+fs_racer_file_link.sh $DIR $MAX_FILES &
+fs_racer_file_link.sh $DIR $MAX_FILES &
+fs_racer_file_link.sh $DIR $MAX_FILES &
 
-./fs_racer_file_symlink.sh $DIR $MAX_FILES &
-./fs_racer_file_symlink.sh $DIR $MAX_FILES &
-./fs_racer_file_symlink.sh $DIR $MAX_FILES &
+fs_racer_file_symlink.sh $DIR $MAX_FILES &
+fs_racer_file_symlink.sh $DIR $MAX_FILES &
+fs_racer_file_symlink.sh $DIR $MAX_FILES &
 
-./fs_racer_file_concat.sh $DIR $MAX_FILES &
-./fs_racer_file_concat.sh $DIR $MAX_FILES &
-./fs_racer_file_concat.sh $DIR $MAX_FILES &
+fs_racer_file_concat.sh $DIR $MAX_FILES &
+fs_racer_file_concat.sh $DIR $MAX_FILES &
+fs_racer_file_concat.sh $DIR $MAX_FILES &
 
-./fs_racer_file_list.sh $DIR &
-./fs_racer_file_list.sh $DIR &
-./fs_racer_file_list.sh $DIR &
+fs_racer_file_list.sh $DIR &
+fs_racer_file_list.sh $DIR &
+fs_racer_file_list.sh $DIR &
 
-./fs_racer_file_rm.sh $DIR $MAX_FILES &
-./fs_racer_file_rm.sh $DIR $MAX_FILES &
-./fs_racer_file_rm.sh $DIR $MAX_FILES &
+fs_racer_file_rm.sh $DIR $MAX_FILES &
+fs_racer_file_rm.sh $DIR $MAX_FILES &
+fs_racer_file_rm.sh $DIR $MAX_FILES &
 }
 
 
@@ -78,14 +78,7 @@ usage()
 call_exit()
 {
     echo \"Cleaning up\"
-    killall fs_racer_file_create.sh
-    killall fs_racer_dir_create.sh
-    killall fs_racer_file_rm.sh
-    killall fs_racer_file_rename.sh
-    killall fs_racer_file_link.sh
-    killall fs_racer_file_symlink.sh
-    killall fs_racer_file_list.sh
-    killall fs_racer_file_concat.sh
+    kill `pgrep fs_racer_`
     rm -rf $DIR
     exit 0
 }
